deploy/manifests/emqx-operator-controller.yaml

```diff
@@ -8745,19 +8717,6 @@ subjects:
   name: emqx-operator-controller-manager
   namespace: emqx-operator-system
 ---
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: emqx-operator-proxy-rolebinding
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: emqx-operator-proxy-role
-subjects:
-- kind: ServiceAccount
-  name: emqx-operator-controller-manager
-  namespace: emqx-operator-system
----
```

Comment:
Please check if there is the same content in helm template, if so, please delete